docs/en/tutorials/test_results_submission.md

```diff
@@ -0,0 +1,65 @@
+# Tutorial 11: Test Results Submission
+
+## Panoptic segmentation test results submission
+
+The following sections introduce how to produce the prediction results of panoptic segmentation models on the COCO test-dev set and submit the predictions to [COCO's evaluation server](https://competitions.codalab.org/competitions/19507).
+
+### Prerequisites
+
+- Download [COCO test dataset images](http://images.cocodataset.org/zips/test2017.zip), [testing image info](http://images.cocodataset.org/annotations/image_info_test2017.zip), and [panoptic train/val annotations](http://images.cocodataset.org/annotations/panoptic_annotations_trainval2017.zip), then unzip them, put 'test2017' to `data/coco/`, put 'panoptic_val2017.json', 'image_info_test-dev2017.json' and 'image_info_test2017.json' to `data/coco/annotations/`.
```

Comment:
Should show the directory structure so that users can understand the file structure after unzip.